apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -75,13 +69,9 @@ export default class LevelFeedbackEntry extends Component {
             </span>
             <span>{this.props.lessonName}</span>
             <span style={styles.label}>{i18n.feedbackNotificationLevel()}</span>
-            <span>{this.props.levelName}</span>
+            <span>{this.props.levelNum}</span>
           </div>
-          <div style={styles.courseUnit}>
-            <span style={styles.label}>
-              {i18n.feedbackNotificationCourse()}
-            </span>
-            <span>{this.props.courseName}</span>
+          <div>
```

Comment:
i think something is wonky in the indentation here -- the `div` on this line should be lined up with the `span` above it

Reply:
lol nevermind, i just can't read 😂